transactions/src/main/java/org/ehcache/transactions/xa/internal/xml/TxCacheManagerServiceConfigurationParser.java

```diff
@@ -71,11 +83,42 @@ public URI getNamespace() {
 
   @Override
   public Class<TransactionManagerProvider> getServiceType() {
-    return null;
+    return TransactionManagerProvider.class;
   }
 
   @Override
   public Element unparseServiceCreationConfiguration(ServiceCreationConfiguration<TransactionManagerProvider> serviceCreationConfiguration) {
-    throw new UnsupportedOperationException("Not yet implemented");
+    try {
+      validateParameterForTranslationToServiceConfig(serviceCreationConfiguration);
+      Document doc = createDocumentRoot();
+      LookupTransactionManagerProviderConfiguration lookupTransactionManagerProviderConfiguration = (LookupTransactionManagerProviderConfiguration) serviceCreationConfiguration;
+      Element rootElement = createRootElement(doc, lookupTransactionManagerProviderConfiguration);
+      return rootElement;
+    } catch (SAXException | ParserConfigurationException | IOException e) {
```

Comment:
Could we try and carve out which of this exceptions are likely user errors due to the input and which are setup errors?  I'm pretty sure the ParserConfigurationException can't be caused by errors in the user input.